fwdctl/src/check.rs

```diff
@@ -105,7 +108,73 @@ fn calculate_area_totals(area_counts: &BTreeMap<u64, u64>) -> (u64, u64) {
     (total_area_count, total_area_bytes)
 }
 
+const TEMPLATE: &str = r"
```

Comment:
because this is a static template, may I recommend [`askama`](https://docs.rs/askama/latest/askama/) instead of handlebars? `askama` will parse the template at compile time to ensure the parameters and template is valid.

Reply:
a partial example of something I wrote that uses this: https://gist.github.com/demosdemon/d7f1a45332af6a4e83ed4e47943e28ad

the template filename extension is irrelevant, but I used `.jinja2` because of some vscode extensions that provide highlighting.

Reply:
Make sense - I was using `handlebars` over `askama` because I don't want to create another report struct type for the report. Do you think it makes more sense to just `impl Display` for the `db_stats` type?

Reply:
Yeah, that is also a good idea. A custom display implementation is common for this especially since it's very straightforward.

Reply:
I decided to move to `askama` instead of `impl Display` to avoid including `num-format` in `storage` crate.